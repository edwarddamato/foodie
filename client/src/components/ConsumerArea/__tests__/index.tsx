import * as React from 'react';
import { ConsumerArea } from '../';
import { create } from 'react-test-renderer';

it('renders correctly', () => {
  const consumerArea = create(
    <ConsumerArea />
  ).toJSON();
  expect(consumerArea).toMatchSnapshot();
});