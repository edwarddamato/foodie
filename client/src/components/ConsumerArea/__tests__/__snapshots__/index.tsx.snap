// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="component_consumer"
>
  <div
    className="form consumer_form"
  >
    <div
      className="form_field consumer_form-field"
    >
      <label
        className="form_field-label"
        htmlFor="txtConsumerName"
      >
        Your Name:
      </label>
      <input
        className="form_field-input"
        id="txtConsumerName"
        onChange={[Function]}
        placeholder="E.g. Bob Smith"
        type="text"
        value=""
      />
    </div>
  </div>
</section>
`;
